utils.go

```diff
@@ -2,6 +2,7 @@ package onepassword
 
 import (
 	"fmt"
+	"math/rand"

```

Comment:
Use `crypto/rand` to avoid weak G404: Use of weak random number generator  https://github.com/securego/gosec/issues/294

Reply:
Done!